modules/nw-ptp-configuring-linuxptp.adoc

```diff
@@ -46,12 +47,13 @@ spec:
 <5> Specify the network interface name to use by the `ptp4l` service, for example `ens787f1`.
 <6> Specify system config options for the `ptp4l` service, for example `-s -2`. This should not include the interface name `-i <interface>` and service config file `-f /etc/ptp4l.conf` because these will be automatically appended.
 <7> Specify system config options for the `phc2sys` service, for example `-a -r`.
-<8> Specify an array of one or more `recommend` objects which define rules on how the `profile` should be applied to nodes.
-<9> Specify the `profile` object name defined in the `profile` section.
-<10> Specify the `priority` with an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`. If a node can be matched with multiple profiles according to rules defined in the `match` field, the profile with the higher priority will be applied to that node.
-<11> Specify `match` rules with `nodeLabel` or `nodeName`.
-<12> Specify `nodeLabel` with the `key` of `node.Labels` from the node object by using the `oc get nodes --show-labels` command.
-<13> Specify `nodeName` with `node.Name` from the node object by using the `oc get nodes` command.
+<8> Specify a string that contains the configuration to replace the default /etc/ptp4l.conf file. To use the default configuration, leave the field empty.
+<9> Specify an array of one or more `recommend` objects which define rules on how the `profile` should be applied to nodes.
```

Comment:
```suggestion
<9> Specify an array of one or more `recommend` objects, which define rules on how the `profile` should be applied to nodes.
```

Reply:
See ISG for reference: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases